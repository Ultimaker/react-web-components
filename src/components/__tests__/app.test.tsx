// Copyright (c) 2018 Ultimaker B.V.
import * as React from 'react';
import { shallow } from 'enzyme';

// component
import App from '../app';

describe('The App component', () => {
    let props;
    let wrapper;

    beforeEach(() => {
        props = {
            fixedHeader: true,
        };
        wrapper = shallow(<App {...props} />);
    });

    it('should render', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
