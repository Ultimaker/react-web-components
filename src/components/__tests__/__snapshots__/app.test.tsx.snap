// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The App component should render 1`] = `
<div
  className="app app--fixed-header"
/>
`;
